README.md

```diff
@@ -1 +1,77 @@
-# wip-platform-be
+# WIP Platform Backend
+
+## Prerequisites
+
+Ensure that Docker and Java 21 are installed on your system.
+
+## Setup
+
+Clone repository (open with Intelij for development).
+
+Start container that launches database in your local environment using `run_db.sh` for Linux/MacOS or `run_db.bat` for Windows.
+
+Launch the entire project by executing `run_project.sh` for Linux/MacOS or `run_project.bat` for Windows.
+
+## General rules
+
+When writing code we follow the SOLID principles and try to keep the code clean.
+
+More on SOLID: https://wearecommunity.io/communities/epam-poland/articles/1190
+
+More on how to keep the code clean: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29
+
+**Follow the UML diagrams from Miro, in case of missing features contact people in charge of the project/backend.**
+
+## Project Structure
+
+We use the following project structure
+
+![image](https://github.com/WIP-Work-In-Progress/wip-platform-be/assets/116646363/5010d8c3-0528-426f-9166-057902cb251a)
```

Comment:
Could you please translate this model for GitHub docs for general language consitency?